src/main/java/org/gridsuite/odre/server/utils/DistanceCalculator.java

```diff
@@ -12,6 +12,7 @@
 /**
  * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
  */
+@SuppressWarnings("checkstyle:HideUtilityClassConstructor")
```

Comment:
should fix the warning instead of suppressing it in this case

Reply:
same remark